### ISSUE TITLE
Week numbers displayed incorrectly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bardsoftware/ganttproject#136]
```
Hi, bug supposed to fixed with 2.0.4 release but still an issue in 2.0.10. Days 
are shown correct when zooming in.
```

Original issue reported on code.google.com by `melm...@gmail.com` on 17 Aug 2010 at 1:24

Attachments:
- [weeknrs.jpg](https://storage.googleapis.com/google-code-attachments/ganttproject/issue-136/comment-0/weeknrs.jpg)

**Comment by syngrato** [comment]
Week numbers are off by one if compared to the [ISO standard](https://en.wikipedia.org/wiki/ISO_week_date). I am using GanttProject 2.7.1 Ostrava (build 1924) on OS X.
![screen shot 2016-01-27 at 14 50 24](https://cloud.githubusercontent.com/assets/16685053/12615328/5db0b4b8-c505-11e5-9811-ec49cce010a2.png)

**Comment by dbarashev** [comment]
Please read this: http://forum.ganttproject.biz/view/20150817124654_27ziajhopwjtxserosos6mtui

TL;DR: switch interface language to English (United Kingdom)

**Comment by syngrato** [comment]
On 2016-01-27 15:05, Dmitry Barashev wrote:
> Please read this:
> http://forum.ganttproject.biz/view/20150817124654_27ziajhopwjtxserosos6mtui
> [1]
> 
> TL;DR: switch interface language to English (United Kingdom)
> 
> --
> Reply to this email directly or view it on GitHub [2].
> 
> 	*
> 
> Links:
> ------
> [1] 
> http://forum.ganttproject.biz/view/20150817124654_27ziajhopwjtxserosos6mtui
> [2]
> https://github.com/bardsoftware/ganttproject/issues/136#issuecomment-175646218
Perfect, thanks.
